tools/remove_from.hpp

```diff
@@ -12,63 +12,60 @@
 namespace antler {
    struct remove_from_project {
 
-      template <antler::project::object::type_t Ty>
+      template <typename Obj>
       bool remove_obj(antler::project::project& proj) {
          if (obj_name.empty()) {
-            std::cerr << "Name is empty" << std::endl;
+            system::error_log("Object name is empty.");
             return false;
          }
 
-         if (proj.remove(obj_name, Ty)) {
-            std::cout << "Removing object: " << obj_name << " from the project." << std::endl;
+         if (proj.remove(obj_name)) {
+            system::info_log("Object {0} removed from the project.", obj_name);
             return true;
          }
          return false;
       }
 
-      inline bool remove_app(antler::project::project& proj) { return remove_obj<antler::project::object::type_t::app>(proj); }
-      inline bool remove_lib(antler::project::project& proj) { return remove_obj<antler::project::object::type_t::lib>(proj); }
-      inline bool remove_test(antler::project::project& proj) { return remove_obj<antler::project::object::type_t::test>(proj); }
+      inline bool remove_app(antler::project::project& proj) { return remove_obj<antler::project::app_t>(proj); }
+      inline bool remove_lib(antler::project::project& proj) { return remove_obj<antler::project::lib_t>(proj); }
+      //inline bool remove_test(antler::project::project& proj) { return remove_obj<antler::project::test_t>(proj); }

```

Comment:
my understanding is that test is not working yet so you commented this out? it would be nice to have TODO section to not mess this with debug or unnecessary code